README.md

```diff
@@ -0,0 +1,53 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List 
+
+> This is an interactive tool to update and remind yourself of what you need to do.
+
+![screenshot](./img/screen_shot.png)
+
+## Built With
+
+- JavaScript
+- Webpack
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://priceless-goldstine-d252ba.netlify.app/)
```

Comment:
- [ ] Your live demo link is currently broken. I cannot access the linked page. Please correct this or remove it from your README file since it is not necessary. This is to make your repo look professional.